api/scripts/normalize_all_addresses.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
```

Comment:
can you please name this file "find_unnormalized_transfer_addresses.py" and also could you add an outputFile param (copy the create_gallery_json_from_static.py script)